lib/ui/painting/fragment_shader.cc

```diff
@@ -34,15 +34,16 @@ std::shared_ptr<DlColorSource> FragmentShader::shader(
   return source_;
 }
 
-fml::RefPtr<FragmentShader> FragmentShader::Create(Dart_Handle dart_handle,
-                                                   sk_sp<SkShader> shader) {
+fml::RefPtr<FragmentShader> FragmentShader::Create(
+    Dart_Handle dart_handle,
+    std::shared_ptr<DlColorSource> shader) {
```

Comment:
Should we tighten this to a RuntimeEffectColorSource?

Reply:
Done